lib/styled.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+let cli   = require('..');
+
+String.prototype.toTitleCase = function() {
```

Comment:
Yeah, saw the linter complain there... I tried not to avoid it, but it got so inelegant. What's best practice here?